.github/workflows/main.yml

```
@@ -0,0 +1,59 @@
+env:
+  RUBY_VERSION: 2.3.7
+  FLOW_TOKEN: testflow
+
+name: Code Quality
+
+on: [push]
+
+jobs:
+  rspec-test:
+    name: RSpec
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres:11.6
+        ports: ["5432:5432"]
+        env:
+          POSTGRES_USER: ${{ secrets.POSTGRES_USER }}
+          POSTGRES_PASSWORD: ${{ secrets.POSTGRES_PASSWORD }}
+          POSTGRES_DB: ${{ secrets.POSTGRES_DB }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/cache@v1  
+        with: 
+          key: ${{ runner.os }}-gems-${{ hashFiles('**/Gemfile.lock') }}  
+          path: vendor/bundle
+      - uses: ruby/setup-ruby@master
+        id: ruby
+        with:
+          ruby-version: ${{ env.RUBY_VERSION }}
+      - name: Install postgres client
+        run: sudo apt-get install libpq-dev
+      - name: Install dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install libcurl3-dev
+          gem install bundler -v 1.10.6
```

Maybe we could try a modern bundler version?

```suggestion
          gem install bundler
```